src/term/mod.rs

```diff
@@ -252,6 +252,10 @@ impl Tag {
   pub fn list_lcons_head() -> Self {
     Self::Named(Name::new("List.LCons.head"))
   }
+
+  pub fn list_lcons_tail() -> Self {
+    Self::Named(Name::new("List.LCons.tail"))
```

Comment:
I think we could improve this functions to use the constants instead of magic strings, like Name(format!("{LIST}.{LCONS}.{HEAD}")). Could you update this now?